File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1150.cs
using System;

class URI
{
    static void Main() {
        int x, z, soma=0, i=0;
        x = int.Parse(System.Console.ReadLine().Trim());
        while(true){
            z = int.Parse(System.Console.ReadLine().Trim());
            if(z>x){
                break;
            }
        }
        while(soma<z){
            soma+=i+x;
            i++;
        }
        Console.Write("{0}\n", i);
        
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1150.cpp
#include <iostream>

using namespace std;

int main(){
    int x, z, soma=0, i=0;
    cin >> x;
    while(true){
        cin >> z;
        if(z>x){
            break;
        }
    }
    while(soma<z){
        soma+=i+x;
        i++;
    }
    cout << i << endl;

    return 0;
}